drivers/sensor/lsm6dsl/lsm6dsl_trigger.c

```diff
@@ -107,8 +127,8 @@ int lsm6dsl_init_interrupt(struct device *dev)
 	}
 
 	gpio_pin_configure(drv_data->gpio, DT_INST_0_ST_LSM6DSL_IRQ_GPIOS_PIN,
-			   GPIO_DIR_IN | GPIO_INT | GPIO_INT_EDGE |
-			   GPIO_INT_ACTIVE_HIGH | GPIO_INT_DEBOUNCE);
+			   GPIO_INPUT | GPIO_INT_DEBOUNCE
```

Comment:
We could remove `GPIO_INT_DEBOUNCE`.

Reply:
I propose to remove `GPIO_INT_DEBOUNCE` flag. We're dealing here with a 'high' quality signal produced by the LSM6DSL.

Reply:
Done